plugin/src/main/kotlin/com/louiscad/splitties/PluginConfig.kt

```diff
@@ -5,4 +5,6 @@ object PluginConfig {
     const val GROUP = "com.louiscad.splitties"
     const val GRADLE_PROPERTY = "version.$GROUP"
     const val SPLITTIES_VERSION = "3.0.0-alpha06"
+
+    var ALREADY_RUN: Boolean = false
```

Comment:
@jmfayard Shouldn't this be `internal`?
```suggestion
    internal var ALREADY_RUN: Boolean = false
```

Reply:
it can be yes